tests/unit/archive_validate_test.py

```diff
@@ -299,3 +306,160 @@ def test_run_summary_success(specs, expected_success):
+                }
+            ],
+            "resources": [
+                {
+                    "profile": "data-resource",
+                    "name": "epacems-1995.zip",
+                    "path": "https://zenodo.org/records/10306114/files/epacems-1995.zip",
+                    "remote_url": "https://zenodo.org/records/10306114/files/epacems-1995.zip",
+                    "title": "epacems-1995.zip",
+                    "parts": {
+                        "year_quarter": [
+                            "1995q1",
+                            "1995q2",
+                            "1995q3",
+                            "1995q4",
+                        ]
+                    },
+                    "encoding": "utf-8",
+                    "mediatype": "application/zip",
+                    "format": ".zip",
+                    "bytes": 18392692,
+                    "hash": "f4ab7fbaa673fa7592feb6977cc41611",
+                },
+                {
+                    "profile": "data-resource",
+                    "name": "epacems-1996.zip",
+                    "path": "https://zenodo.org/records/10306114/files/epacems-1996.zip",
+                    "remote_url": "https://zenodo.org/records/10306114/files/epacems-1996.zip",
+                    "title": "epacems-1996.zip",
+                    "parts": {
+                        "year_quarter": [
+                            "1996q1",
+                            "1996q2",
+                            "1996q3",
+                            "1996q4",
+                        ]
+                    },
+                    "encoding": "utf-8",
+                    "mediatype": "application/zip",
+                    "format": ".zip",
+                    "bytes": 18921381,
+                    "hash": "fb35601de4e849a6cd123e32f59623f9",
+                },
+                {
+                    "profile": "data-resource",
+                    "name": "epacems-1997.zip",
+                    "path": "https://zenodo.org/records/10306114/files/epacems-1997.zip",
+                    "remote_url": "https://zenodo.org/records/10306114/files/epacems-1997.zip",
+                    "title": "epacems-1997.zip",
+                    "parts": {
+                        "year_quarter": [
+                            "1997q1",
+                            "1997q2",
+                        ]
+                    },
+                    "encoding": "utf-8",
+                    "mediatype": "application/zip",
+                    "format": ".zip",
+                    "bytes": 21734463,
+                    "hash": "e9ece6bb8190e14d44834bf0a068ac5d",
+                },
+            ],
+            "created": "2023-11-30 20:51:43.255388",
+            "version": "1",
+        }
+    )
+)
+fake_new_datapackage_month_success = copy.deepcopy(fake_new_datapackage_quarter_success)
+# Make a successful monthly datapackage based on the quarterly one
```

Comment:
We can add an `id` field to the `pytest.mark.parameterize` with a list of test IDs here rather than comments, so we know what each test is. I just did this in https://github.com/catalyst-cooperative/pudl-archiver/pull/236 if you need an example.